comparison.md

```diff
@@ -77,6 +77,6 @@ support.
 > To get a free trial of Coder, please visit
 > [https://coder.com/trial](https://coder.com/trial).
 
-> Coder's trial license does not work in an air-gapped environment. If your
-> organization is interested in evaluating Coder air-gapped, please contact
-> [sales@coder.com](mailto:sales@coder.com) to discuss license requirements.
+Coder's trial license does not work in an air-gapped environment. If your
+organization is interested in evaluating Coder air-gapped, please contact
+[sales@coder.com](mailto:sales@coder.com) to discuss license requirements.
```

Comment:
0cffb5081daf30820f041d94ebf49782f22362ae introduced a linter warning that caused tests to fail